cattqt/cattqt.py

```diff
@@ -676,6 +676,7 @@ def on_play_next(self, d):
         self.textbox.setText(text)
 
     def on_start_singleshot_timer(self, d):
+        d.playback_starting = False
```

Comment:
What is this,

Reply:
Ok, I patched from the 3.0 tag. That answers the question I just asked.